lib/visualstudio.js

```diff
@@ -219,6 +220,9 @@ function detectInstallations(emitter, options, callback) {
 
 						// verify that this Visual Studio actually exists
 						if (info.path && fs.existsSync(info.path)) {
+
+							info.vsDevCmd = path.join(info.path, 'Common7', 'Tools', 'VsDevCmd.bat');
```

Comment:
This lIne 224 is needed when you use previous version of Visual Studio (2013 and 2015). Line 120 is used only when it detects Visual Studio 2017.